Makefile

```diff
@@ -479,15 +479,16 @@ BUNDLE_IMG ?= $(PROJECT)-bundle:v$(VERSION)
 BUNDLE_OPERATOR_MANIFEST ?= deploy/operator.yaml
 
 # These examples are added to the alm-examples annotation and subsequently
-# displayed in the UI
+# displayed in the UI. Keep the separator last.
 OLM_EXAMPLES := \
 	examples/apparmorprofile.yaml \
 	examples/config.yaml \
 	examples/profilerecording-seccomp-bpf.yaml \
 	examples/profilebinding.yaml \
 	examples/rawselinuxprofile.yaml \
 	examples/seccompprofile.yaml \
-	examples/selinuxprofile.yaml
+	examples/selinuxprofile.yaml \
+	deploy/separator.yaml
```

Comment:
so this is clearly a hack, but I didn't figure out how could I ask `kustomize` to produce `operator.yaml` with separators on the first line.

Anyone has an idea?

Reply:
I don't let's
/hold 
and feel free to unhold if there is no other solution in sight.